Cargo.toml

```diff
@@ -50,6 +50,7 @@ async-lock = "2.6"
 # Benchmark only dependencies
 clap = { version = "3.2", optional = true }
 env_logger = { version = "0.9", optional = true }
+rusty-fork = "0.3.0"
```

Comment:
This dependency should be within `dev-dependencies`, because it is only needed for tests